parm/config/config.resources.nco.static

```diff
@@ -49,89 +49,99 @@ if [ $step = "prep" -o $step = "prepbufr" ]; then
+    export NTASKS=$npe_waveprep
+    export NTASKS_gfs=$npe_waveprep_gfs
 
 elif [ $step = "wavepostsbs" ]; then
 
-    export wtime_wavepostsbs="06:00:00"
-    export npe_wavepostsbs=10
+    export wtime_wavepostsbs="00:20:00"
+    export wtime_wavepostsbs_gfs="03:00:00"
+    export npe_wavepostsbs=8
     export nth_wavepostsbs=1
-    export npe_node_wavepostsbs=$(echo "$npe_node_max / $nth_wavepostsbs" | bc)
-    export NTASKS=${npe_wavepostsbs}
+    export npe_node_wavepostsbs=$npe_wavepostsbs
+    export memory_wavepostsbs="1GB"
+    export NTASKS=$npe_wavepostsbs
 
 elif [ $step = "wavepostbndpnt" ]; then
 
     export wtime_wavepostbndpnt="01:00:00"
     export npe_wavepostbndpnt=448
     export nth_wavepostbndpnt=1
     export npe_node_wavepostbndpnt=$(echo "$npe_node_max / $nth_wavepostbndpnt" | bc)
-    export NTASKS=${npe_wavepostbndpnt}
+    export NTASKS=$npe_wavepostbndpnt
 
 elif [ $step = "wavepostbndpntbll" ]; then
 
     export wtime_wavepostbndpntbll="01:00:00"
     export npe_wavepostbndpntbll=448
     export nth_wavepostbndpntbll=1
     export npe_node_wavepostbndpntbll=$(echo "$npe_node_max / $nth_wavepostbndpntbll" | bc)
-    export NTASKS=${npe_wavepostbndpntbll}
+    export NTASKS=$npe_wavepostbndpntbll
 
 elif [ $step = "wavepostpnt" ]; then
 
     export wtime_wavepostpnt="01:30:00"
     export npe_wavepostpnt=448
     export nth_wavepostpnt=1
     export npe_node_wavepostpnt=$(echo "$npe_node_max / $nth_wavepostpnt" | bc)
-    export NTASKS=${npe_wavepostpnt}
+    export NTASKS=$npe_wavepostpnt
 
 elif [ $step = "wavegempak" ]; then
 
-    export wtime_wavegempak="01:00:00"
-    export npe_wavegempak=$npe_node_max
+    export wtime_wavegempak="02:00:00"
+    export npe_wavegempak=1
     export nth_wavegempak=1
-    export npe_node_wavegempak=$(echo "$npe_node_max / $nth_wavegempak" | bc)
-    export NTASKS=${npe_wavegempak}
+    export npe_node_wavegempak=$npe_wavegempak
+    export NTASKS=$npe_wavegempak
+    export memory_wavegempak="1GB"
 
 elif [ $step = "waveawipsbulls" ]; then
 
-    export wtime_waveawipsbulls="00:30:00"
+    export wtime_waveawipsbulls="00:20:00"
     export npe_waveawipsbulls=$npe_node_max
     export nth_waveawipsbulls=1
     export npe_node_waveawipsbulls=$(echo "$npe_node_max / $nth_waveawipsbulls" | bc)
-    export NTASKS=${npe_waveawipsbulls}
+    export NTASKS=$npe_waveawipsbulls
 
 elif [ $step = "waveawipsgridded" ]; then
 
-    export wtime_waveawipsgridded="00:30:00"
+    export wtime_waveawipsgridded="02:00:00"
     export npe_waveawipsgridded=$npe_node_max
     export nth_waveawipsgridded=1
     export npe_node_waveawipsgridded=$(echo "$npe_node_max / $nth_waveawipsgridded" | bc)
-    export NTASKS=${npe_waveawipsgridded}
+    export NTASKS=$npe_waveawipsgridded
 
 elif [ $step = "anal" ]; then
 
-    export wtime_anal="00:45:00"
-    export npe_anal=1000
-    export nth_anal=12
-    export npe_anal_gfs=1000
-    export nth_anal_gfs=12
+    export wtime_anal="00:50:00"
```

Comment:
NCO has 1:00 for gdas and 0:40 for gfs

Reply:
@KateFriedman-NOAA and I spoke.  We will make them both 50 mins in the PR #603